schism/mplink.c

```diff
@@ -222,6 +222,29 @@ int song_find_last_channel(void)
 
 // ------------------------------------------------------------------------
 
+int song_get_pattern_offset(int * n, song_note_t ** buf, int * r, int offset)
+{
+	int len, tot;
+	if (song_get_mode() & (MODE_PATTERN_LOOP)) {
```

Comment:
Not a big deal, but `MODE_PATTERN_LOOP` doesn't need to be parenthesized here.

Reply:
right. fixing.